### ISSUE TITLE
spidersweb.pl nawigacja zniknęła

### ISSUE THREAD

**Issue by spidersweb-header** [MajkiIT/polish-ads-filter#2337]
Po zmianie wyglądu na stronie traci się nawigacja.
Wyłączenie głównego filtru polskiego z reklamami ją przywraca.
Warto sprawdzić co wycina nawigacje.

>**Test z Adblock Plus / Adblock:**
![image](https://cloud.githubusercontent.com/assets/26621269/24239999/5c83d952-0fb0-11e7-88f9-317d7aec2830.png)
![image](https://cloud.githubusercontent.com/assets/26621269/24240219/19079a6e-0fb1-11e7-96ff-ac0564f7e530.png)
**aktywny uBlock:**
![image](https://cloud.githubusercontent.com/assets/26621269/24239857/ce9d1e8c-0faf-11e7-91a3-9b3f2a80b4ab.png)
**Nieaktywny uBlock:**
![image](https://cloud.githubusercontent.com/assets/26621269/24239893/ed97aa28-0faf-11e7-9f04-23b5024bb8bf.png)

uBlock 1.1.14 na Chromium / Firefox; Adblock Plus 1.13.2 / Adblock 3.8.8 na Chromium

**Comment by got3n** [comment]
Potwierdzam, właśnie miałem to raportować.

**Comment by F4z** [comment]
Dzięki za zgłoszenie.
Tak, to wina tej reguły - `spidersweb.pl##.text-center`

@MajkiIT usunę tę regułę - `spidersweb.pl##.text-center` - i dodam te dwie:
`spidersweb.pl##.sidebar-panel[data-equalizer-watch^="entrieshome"][id^="sidebar_anchor_parent_"]`
`spidersweb.pl##a[href*="&utm_medium=baner"][target] > img[src^="http://ocs-pl.oktawave.com/"]:not([class])`

Można by jeszcze zablokować konkretne obrazki aby mniej danych było pobieranych przy odwiedzaniu strony:
```
||ocs-pl.oktawave.com/v1/AUTH_2887234e-384a-4873-8bc5-405211db13a2/spidersweb/2017/03/MV_nolimit_1180x200.jpg
||ocs-pl.oktawave.com/v1/AUTH_2887234e-384a-4873-8bc5-405211db13a2/spidersweb/2017/03/EXPLORE_1401_Spidersweb_3*x*px_V*_02-03-2017.jpg
||ocs-pl.oktawave.com/v1/AUTH_2887234e-384a-4873-8bc5-405211db13a2/spidersweb/2017/03/300x700.jpg
||ocs-pl.oktawave.com/v1/AUTH_2887234e-384a-4873-8bc5-405211db13a2/spidersweb/2017/03/400x400-1.jpg
||ocs-pl.oktawave.com/v1/AUTH_2887234e-384a-4873-8bc5-405211db13a2/splay/2017/03/xc_laser_400x400_2.jpg
||ocs-pl.oktawave.com/v1/AUTH_2887234e-384a-4873-8bc5-405211db13a2/splay/2017/03/xc_laser_300x700_2.jpg
||ocs-pl.oktawave.com/v1/AUTH_2887234e-384a-4873-8bc5-405211db13a2/spidersweb/2017/03/1200x300.jpg
```
Ale z tego co widzę, to oni chyba co miesiąc zmieniają te reklamy - https://web.archive.org/web/*/spidersweb.pl - więc nie wiem czy jest sens się w to bawić.
Ewentualnie można dodać to do sekcji "Rules temporary blocking filters" (bo to sekcja na tymczasowe reguły, dobrze rozumiem?) i co miesiąc zmieniać, jeśli będzie taka potrzeba.

**Comment by xxcriticxx** [comment]
nie lepej zablokowac cale `oktwave.com`

**Comment by F4z** [comment]
@xxcriticxx nie, bo zablokujesz nie tylko reklamy, ale też inne obrazki, na przykład - http://ocs-pl.oktawave.com/v1/AUTH_2887234e-384a-4873-8bc5-405211db13a2/spidersweb/2017/03/nowy-spidersweb-780x510.png

**Comment by xxcriticxx** [comment]
@F4z ok
